projects/igniteui-angular/src/lib/core/styles/components/action-strip/_action-strip-theme.scss

```diff
@@ -107,6 +107,7 @@
                 width: var(--ig-icon-size, $icon-size);
                 height: var(--ig-icon-size, $icon-size);
                 font-size: var(--ig-icon-size, $icon-size);
+                pointer-events: none;
```

Comment:
The extended icons include <Title> which is on an inner element and that causes the default browser hover action to show that, instead of the provided IgxLabel on higher level.